docs/02.API-REFERENCE.md

```diff
@@ -62,6 +62,15 @@ Possible compile time enabled feature types:
  - JERRY_FEATURE_LINE_INFO - line info available
  - JERRY_FEATURE_LOGGING - logging
 
+## jerry_regexp_flags_t
+
+RegExp object optional flags
```

Comment:
Add a colon (:) to the end of the line and also add an empty line between this line and the following list (see other enum documentations for example).